student.rb

```diff
@@ -9,4 +9,9 @@ def initialize(age, classroom, name = 'Unknown', parent_permission: true)
   def play_hooky
     '¯\(ツ)/¯'
   end
+
+  def add_classroom(classroom)
+    @classroom = classroom
+    classroom.students << self unless classroom.students.include?(self)
+  end
```

Comment:
- [ ] When setting the classroom for a student it also adds it to the classrooms' students. You're missing an accessor for `classroom`

Reply:
well resolved! ✔️